ui/raidboss/data/07-dt/raid/r4n.ts

```diff
@@ -114,6 +171,25 @@ const headMarkerData = {
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM4',
   zoneId: ZoneId.AacLightHeavyweightM4,
+  config: [
+    {
+      id: 'sidewiseSpark',
+      name: {
+        en: 'Sidewise Spark Clone Callout Method',
+      },
+      comment: {
+        en: 'Select how to callout Sidewise Spark cleaves from clones.',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Sequence of moves as cleaves finish': 'sequence',
+          'Collect all cleaves and call once': 'collected',
+        },
+      },
+      default: 'collected',
+    },
```

Comment:
I think we should standardize on always listing the default option first for trigger config options.
```suggestion
      options: {
        en: {
          'Collect all cleaves and call once': 'collected',
          'Sequence of moves as cleaves finish': 'sequence',
        },
      },
      default: 'collected',
    },
```
I think we should standardize on always listing the default option first for trigger config options.